_service

```diff
@@ -4,7 +4,7 @@
     <param name="scm">git</param>
     <param name="exclude">.git</param>
     <param name="filename">saphanabootstrap-formula</param>
-    <param name="versionformat">0.6.2+git.%ct.%h</param>
+    <param name="versionformat">0.6.3+git.%ct.%h</param>
```

Comment:
@Simranpal Let's try to avoid increasing the version here. It will be better we just increase when we are going to deliver and created a new submission.
We can group this way many changes in a new release